applications/app/controllers/CrosswordsController.scala

```diff
@@ -107,7 +107,7 @@ class CrosswordPageController(val contentApiClient: ContentApiClient, val contro
   def printableCrossword(crosswordType: String, id: Int): Action[AnyContent] =
     Action.async { implicit request =>
       withCrossword(crosswordType, id) { (crossword, content) =>
-        Cached(3.days)(
+        Cached(60)(
```

Comment:
Minutes, seconds, milliseconds?

Reply:
seconds - I'm in an unfamiliar code base so chose to match the [surrounding](https://github.com/guardian/frontend/blob/61bb0249a418039306688545eb1bab604ab1b16f/applications/app/controllers/CrosswordsController.scala#L90) [convention](https://github.com/guardian/frontend/blob/61bb0249a418039306688545eb1bab604ab1b16f/applications/app/controllers/CrosswordsController.scala#L130), but if the explicit `.seconds` is preferred I'd be happy to also apply that to those instances too while I'm here

Reply:
Yeah either way I was just unable to tell from the code review alone.